.changes/unreleased/Fixed-20240731-061239.yaml

```
@@ -0,0 +1,9 @@
+kind: Fixed
+body: |
+    feat: extend multi git server support to Azure devOps
+
+    Our support for Azure refs is broken on monorepos, as Azure returns a 500 when we query a ref containing a subpath with `go-get=1`. Instead, we rely on regexes to infer the root of the repo and the subpath from the ref
```

```suggestion
    Our support for Azure refs was broken on monorepos, as Azure returns a 500 when we query a ref containing a subpath with `go-get=1`. Instead, we rely on regexes to infer the root of the repo and the subpath from the ref
```

I don't know if we need the detail about 500s, we can just say we special-case azure dev-ops since it doesn't work with the go standard convention of discovering the root of a git repository.

---

Changed, you're right, it is better 🙏